### PR TITLE
7394: treat apostrophes in words as part of the token

### PR DESCRIPTION
**Pull request by kkoskelin**
very specific character blacklist to be excluded from strings
ustaxcourt/ef-cms#7588

### CODE REVIEW

**Review by mrinsin — APPROVED** [review]
Should we mark this pr as 'Don't Merge' until a new sprint PR has been made? (That's what Rae, Rachael and myself have been doing!)

File: shared/src/business/utilities/aggregateCommonQueryParams.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ const aggregateCommonQueryParams = ({
   const nonExactMatchesQuery = [];
 
   if (petitionerName) {
-    const simplePetitionerQuery = makeSimpleQuerySafe(petitionerName);
+    const simplePetitionerQuery = removeAdvancedSyntaxSymbols(petitionerName);
```
Comment (mrinsin) [inline shared/src/business/utilities/aggregateCommonQueryParams.js:33]: I wonder if we need to use this in `advancedDocumentSerach.js` as well?

Comment (kkoskelin) [reply shared/src/business/utilities/aggregateCommonQueryParams.js:33]: I had neglected to search for it elsewhere. will fix asap.

Comment (mrinsin) [reply shared/src/business/utilities/aggregateCommonQueryParams.js:33]: alt-text

**Pull request merged**